pkg/dockerregistry/server/signaturedispatcher_test.go

```diff
@@ -57,6 +57,8 @@ func TestSignatureGet(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	os.Setenv("OPENSHIFT_DEFAULT_REGISTRY", "localhost:5000")
```

Comment:
@dmage for some strange reasons this is needed (at least on OSX) otherwise the NewApp() will fail... we should make this a field in openshift middleware config

Reply:
Can you fix this as part of this PR? This is pretty wonky

Reply:
@mfojtik yes, configuration check is moved to the app constructor. Adding a field in config will introduce yet another one environment variable for this value (we already have DEFAULT_REGISTRY_URL and OPENSHIFT_DEFAULT_REGISTRY). But I like this idea.

Reply:
fixed in a separate commit